pyimzml/ImzMLWriter.py

```diff
@@ -309,18 +299,18 @@ def main(argv):
     try:
         opts, args = getopt.getopt(argv,"hi:o:",["ifile=","ofile="])
     except getopt.GetoptError:
-        print 'test.py -i <inputfile> -o <outputfile>'
+        print('test.py -i <inputfile> -o <outputfile>')
```

Comment:
It's not needed for python2 compatibility (I use python2), but I'll add it anyway.